src/pages/showcase/submit.astro

```diff
@@ -0,0 +1,190 @@
+]
+
+const toolOptions = [
+    'AlpineJS',
+    'Lit',
+    'MDX',
+    'PostCSS',
+    'Preact',
+    'React',
+    'SASS',
+    'SolidJS',
+    'Svelte',
+    'Tailwind',
+    'TypeScript',
+    'Vue'
+]
+---
+
+<Base title="Submit a theme">
+    <PageTitle>
+        <Fragment slot="title">Submit a theme</Fragment>
+        Thank you for sharing your project with the Astro community! Please fill
+        out this form as completely as possible.
+    </PageTitle>
+
+    <section class="max-w-screen-sm mx-auto px-4 pb-4 pt-12 md:pt-16 lg:pt-20">
+        <header class="mb-4">
+            <h2 class="text-3xl font-semibold pb-2">Best practices</h2>
+            <p class="text-neutral-500">
+                What we look for when reviewing submissions:
+            </p>
+        </header>
+        <ul
+            class="flex flex-col gap-2 mt-6 mb-4 whitespace-normal text-neutral-700 -ml-1"
+        >
+            {
+                bestPractices.map((text) => (
+                    <li class="flex items-center gap-1">
+                        <Icon
+                            name="heroicons-outline:check"
+                            size={28}
+                            class="text-green-500"
+                            aria-hidden="true"
+                        />
+                        {text}
+                    </li>
+                ))
+            }
+        </ul>
+    </section>
+
+    <Aside class="max-w-screen-sm text-sm shadow-md w-full mx-auto">
+        <p class="text-justify">
+            <Icon
+                pack="heroicons-outline"
+                name="information-circle"
+                size="1.5em"
+                class="inline-block"
+                aria-hidden="true"
+            />
+            Looking for inspiration for your theme's preview images? Check out our
+            <a
+                href="/resources/image-templates"
+                target="_blank"
+                class="underline font-semibold">Image Templates</a
+            >!
+        </p>
+    </Aside>
+
+    <form
+        name="themeSubmit"
+        method="POST"
+        enctype="multipart/form-data"
+        action="/.netlify/functions/submit-showcase"
+        class="max-w-screen-sm mx-auto my-8 px-4"
+    >
+        <div class="flex flex-col gap-6 items-center">
+            {/* https://docs.netlify.com/forms/spam-filters/#honeypot-field */}
+            <p class="hidden">
+                <label>
+                    Don&apos;t fill this out if you&apos;re human:{' '}
+                    <input name="botField" />
+                </label>
+            </p>
+
+            <Field label="Preview images (max 8MB combined)">
+                <ImageInput
+                    name="images"
+                    label="Preview image"
+                    required
+                    client:idle
+                />
+
+                <div class="grid md:grid-cols-2 w-full gap-4 pt-3">
```

Comment:
We only actually collect one image for showcase, we can remove the additional image inputs and update the field label to "Preview image" or similar